### PR TITLE
Reportback final cleanup

### LINKED ISSUE #4107: Remove original Reportback Form code
Remove anything we don't need/use anymore. Here is one instance: 

https://github.com/DoSomething/dosomething/blob/v0.4.21/lib/modules/dosomething/dosomething_reportback/dosomething_reportback.theme.inc#L12

### PR DESCRIPTION
**Pull request by weerd**
### Fixes #4107

Removes old code no longer in use now that Reportback v2 has launched.

@aaronschachter @DFurnes

### CODE REVIEW

**Comment by DFurnes** [comment]
:+1: from me!

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php [hunk 1]
```diff
@@ -263,7 +263,7 @@
 
     <div class="wrapper">
       <div class="container__block">
-        <h3 class="heading -beta inline-alt-text-color"><?php print t('Pics or It Didn’t Happen'); ?></h3>
+        <h3 class="heading -beta inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php:266]: Not a biggie, but what's the reason behind this change? Kinda makes translation a little more difficult.

**Comment by aaronschachter** [comment]
Looks good except I think encoding the apostrophe makes things less readable (and not sure why we'd need to encode it)

**Comment by weerd** [comment]
So, it's a right apostrophe. Not a normal one, and always best to encode those. While not as readable it's in general better and actually makes it's more obvious that it's a special apostrophe character. It's not even available on the keyboard. It's a special HTML entity.

**Comment by aaronschachter** [comment]
Dang, you fancy. Alright. :+1:
